api/descriptions/defenders/fargate_yaml_post.md

```diff
@@ -0,0 +1,36 @@
+Returns a protected Fargate task definition given an unprotected task definition.
+
+### cURL Request
+Refer to the following example cURL command:
+
+`<HOSTNAME>` is a single list item from the `/api/v<VERSION>/defenders/names` endpoint.
+
+Unprotected task definition in `unprotected.yaml`
```

Comment:
@ssugandh I'll need to check with Dev if this endpoint is relevant for native fargate task or only for CloudFormation, will get back to you on that